cmake/Modules_CUDA_fix/upstream/FindCUDA/select_compute_arch.cmake

```
@@ -93,7 +93,7 @@ if(CUDA_VERSION VERSION_GREATER "10.5")
   endif()
 endif()
 
-if(CUDA_VERSION VERSION_GREATER "11.0")
+if(CUDA_VERSION VERSION_GREATER "11.0.3")
```

Why not use `VERSION_GREATER_EQUAL` and keep version to just major/minor?
```suggestion
if(CUDA_VERSION VERSION_GREATER_EQUAL "11.1")
```

---

this doesn't work since our min requirement for cmake is 3.5 - VERSION_GREATER_EQUAL is not introduce until 3.7 
should we bump min requirement for cmake for just this?

---

What about something like this? (I may have the syntax wrong since my CMake knowledge is a bit rusty)
```suggestion
if(NOT CUDA_VERSION VERSION_LESS "11.1")
```
or would that not work (e.g. if versions are not a total order)?